docs/community/.nav.yml

```diff
@@ -0,0 +1,7 @@
+# `.nav.yml` is provided by https://lukasgeiter.github.io/mkdocs-awesome-nav
+nav:
+  - Questions & Help: help.md
+  - Contributing: contributing.md
+  - Development: development.md
+  - Code of Conduct: code_of_conduct.md
+  - Dev Guides: devguides
```

Comment:
All these sections were previously under `Overview` section, except for Dev guides, which were in `Guides`.